Dockerfile

```diff
@@ -1,49 +1,59 @@
-FROM node:10.3-alpine
-LABEL maintainer="https://github.com/verdaccio/verdaccio"
+FROM node:10.3-alpine as builder
 
-RUN apk --no-cache add openssl && \
-    wget -O /usr/local/bin/dumb-init https://github.com/Yelp/dumb-init/releases/download/v1.2.0/dumb-init_1.2.0_amd64 && \
-    chmod +x /usr/local/bin/dumb-init && \
-    apk del openssl && \
-    apk --no-cache add ca-certificates wget && \
+RUN apk --no-cache add openssl ca-certificates wget && \
```

Comment:
I'm not sure, I just left it there. I though it was installed for https support in case the `VERDACCIO_PROTOCOL` is updated, but I'm not sure. In our environment we run verdaccio with plain http and front it with a reverse proxy that performs the ssl termination.